PYTHON_CDM_Conversion_code/utils.py

```diff
@@ -19,12 +19,19 @@
 # Subdaily CDM Lite
 # Paths
 SUBDAILY_QFF_IN_DIR = config.get("Paths", "subdaily_qff_indir")
+DAILY_HEAD_IN_DIR = config.get("Paths", "subdaily_head_indir")
 SUBDAILY_QC_OUT_DIR = config.get("Paths", "subdaily_qc_outdir")
 SUBDAILY_CDMLITE_OUT_DIR = config.get("Paths", "subdaily_cdmlite_outdir")
+SUBDAILY_CDMOBS_OUT_DIR = config.get("Paths", "subdaily_cdmobs_outdir")
+SUBDAILY_CDMHEAD_OUT_DIR = config.get("Paths", "subdaily_cdmhead_outdir")
 
 # Files
 SUBDAILY_CDMLITE_FILE_ROOT = config.get("Filenames", "subdaily_cdmlite_file")
 SUBDAILY_QC_FILE_ROOT = config.get("Filenames", "subdaily_qc_file")
+SUBDAILY_CDMOBS_FILE_ROOT = config.get("Filenames", "subdaily_cdmobs_file")
+SUBDAILY_CDMHEAD_FILE_ROOT = config.get("Filenames", "subdaily_cdmhead_file")
 
-# Station records
+# Station records (note there are two difgferent record needd one for obs and on
```

Comment:
Typos in comment

That's useful - what probably makes sense is then to have a `station_records_obs` and `station_records_header` in the config file, and then `STATION_RECORD_ENTRIES_OBS` and `STATION_RECORD_ENTRIES_HEADER` so it's clear what's what going forwards.

Will break cdm_lite script, so make the edits there too so it works as part of this issue and PR

Reply:
Fixed in 8056aaa